src/sql/parts/dashboard/containers/dashboardGridContainer.component.ts

```diff
@@ -136,7 +136,10 @@ export class DashboardGridContainer extends DashboardTab implements OnDestroy {
 		return rowCount * this.cellHeight + 'px';
 	}
 
-	private convertToNumber(value: string, maxNumber: number): number {
+	private convertToNumber(value: string | number, maxNumber: number): number {
```

Comment:
Here you aren't account for if you user inputs a number as a string, which is fine, but theres nothing in the schema that tells the user that.

Reply:
I changed the schema, is it not how user sees the supported types? 

---
In reply to: [172963793](https://github.com/Microsoft/sqlopsstudio/pull/859#discussion_r172963793) [](ancestors = 172963793)